### ISSUE TITLE
Final peer review - jk2332

### ISSUE THREAD

**Issue by jk2332** [yh854/ORIE4741_project#16]
The report investigates data collected from LOL game plays, and aims to discover insights to factors that might lead to higher probability of winning a game. The dataset used is from Kaggle, with around 1,000,000 rows and 76 features.
This report is about providing insights into game LOL plays with Kaggle data with 1,000,000 data points.
It provides good insight into ups and downs of different models and how they work for this data in particular. It also has good explanation of how to improve this project further and how it can be adapted in reality. Looking from the result, the model seems to be adequately fitted as well.
I think this report can delve more into trying out different loss functions more. Also, trying out different hyper-parameters and explanation to how it affects the performance of decision tree could be nice to be incorporated. Another improvement could be to provide more visualization of graphs and plots for better explanation.
